DebeziumAdapter/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11-jre
+MAINTAINER DevelopmentTeam
+WORKDIR /usr/src/adapter
+COPY ./target/*.jar ./DebeziumAdapter-SNAPSHOT.jar
+CMD ["java","-jar","/usr/src/adapter/DebeziumAdapter-SNAPSHOT.jar"]
```

Comment:
Does the container runs with root user? if yes, please create another user and switch to it before executing.